client/src/App.js

```diff
@@ -1,7 +1,13 @@
 import React from "react";
+import GlobalStyle from "./globalStyles";
```

Comment:
If you name other components in PascalCase, consider to rename the module to `GlobalStyle.js`. 
I renamed the file to `GlobalStyle` (singular).